lambdas/process-journey-event/src/main/java/uk/gov/di/ipv/core/processjourneyevent/ProcessJourneyEventHandler.java

```diff
@@ -227,14 +243,23 @@ private boolean sessionIsNewlyExpired(IpvSessionItem ipvSessionItem) {
     }
 
     @Tracing
-    private Map<IpvJourneyTypes, StateMachine> loadStateMachines(List<IpvJourneyTypes> journeyTypes)
+    private Map<IpvJourneyTypes, StateMachine> loadStateMachines(
+            List<IpvJourneyTypes> journeyTypes,
+            StateMachineInitializerMode stateMachineInitializerMode)
             throws IOException {
         EnumMap<IpvJourneyTypes, StateMachine> stateMachinesMap =
                 new EnumMap<>(IpvJourneyTypes.class);
         for (IpvJourneyTypes journeyType : journeyTypes) {
             stateMachinesMap.put(
-                    journeyType, new StateMachine(new StateMachineInitializer(journeyType)));
+                    journeyType,
+                    new StateMachine(
+                            new StateMachineInitializer(journeyType, stateMachineInitializerMode)));
         }
         return stateMachinesMap;
     }
+
+    private Map<IpvJourneyTypes, StateMachine> loadStateMachines(List<IpvJourneyTypes> journeyTypes)
+            throws IOException {
+        return loadStateMachines(journeyTypes, StateMachineInitializerMode.STANDARD);
```

Comment:
Do we need this? Could we just pass the extra argument at it's usage?

Reply:
Yep, I'll make the change